src/app/components/cds/charts/ListWidget.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ListWidgetItem from './ListWidgetItem';
+import styles from './ListWidget.module.css';
+
+const ListWidget = ({
+  color,
+  items,
+  title,
+}) => (
+  <div className={styles.listWidgetWrapper} style={{ backgroundColor: color }}>
+    <div>
+      {title}
+    </div>
+    <ul>
+      { items.map(i => (
+        <ListWidgetItem
+          itemLink={i.itemLink}
+          itemText={i.itemText}
+          itemValue={i.itemValue}
+          key={i.id}
+        />
+      ))}
+    </ul>
+  </div>
+);
+
+ListWidget.defaultProps = {
+  color: 'var(--color-pink-93)',
+};
+
+ListWidget.propTypes = {
+  color: PropTypes.string,
+  items: PropTypes.arrayOf(PropTypes.shape({ link: PropTypes.string, text: PropTypes.string, value: PropTypes.string })).isRequired,
```

Comment:
Hi @vasconsaurus, the propTypes shape doesn't match the actual data: `itemLink`, `itemText` and `itemValue`

Reply:
Please update this and it should be good to go from my perspective! :smile:

Reply:
🤦‍♀️ sorry! updated!